packages/sysutils/systemd/package.mk

```diff
@@ -103,7 +103,8 @@ PKG_CONFIGURE_OPTS_TARGET="ac_cv_func_malloc_0_nonnull=yes \
                            --with-dbussessionservicedir=/usr/share/dbus-1/services \
                            --with-dbussystemservicedir=/usr/share/dbus-1/system-services \
                            --with-rootprefix=/usr \
-                           --with-rootlibdir=/usr/lib"
+                           --with-rootlibdir=/usr/lib \
+                           --with-default-hierarchy=hybrid"
```

Comment:
it looks like the docker issues are already resolved - see comment here https://bugs.archlinux.org/task/53271
haven't tested but if this is the case better patch/update docker than staying on legacy cgroups v1